Other/MergeShops/SeaBaseHouseMerge.cs

```diff
@@ -41,6 +41,7 @@ public void ScriptMain(IScriptInterface Bot)
     public void BuyAllMerge(string? buyOnlyThis = null, mergeOptionsEnum? buyMode = null)
     {
         AoR.TwilightZone();
```

Comment:
You don't need this, MidnightZone already does TwilightZone if it wasnt completed before, the rest looks good